src/main/java/com/dataloom/sync/SyncApi.java

```diff
@@ -12,12 +13,15 @@
     String BASE               = SERVICE + CONTROLLER;
 
     String ENTITY_SET_ID      = "entitySetId";
+    String SYNC_ID            = "syncId";
 
-    String ENTITY_SET_ID_PATH = "{" + ENTITY_SET_ID + "}";
+    String ENTITY_SET_ID_PATH = "/{" + ENTITY_SET_ID + "}";
+    String SYNC_ID_PATH       = "/{" + SYNC_ID + "}";
     String NEW                = "/new";
```

Comment:
You're using `/latest` as a sort of versioning identifier on the URL. Following the same pattern, I think it makes more sense, from a consistency perspective, to do `/current` instead of `/new`. By that, I mean:

```
@GET( BASE + "/" + ENTITY_SET_ID_PATH )
UUID acquireSyncId( @Path( ENTITY_SET_ID ) UUID entitySetId );

@GET( BASE + "/" + ENTITY_SET_ID_PATH + CURRENT )
UUID getCurrentSyncId( @Path( ENTITY_SET_ID ) UUID entitySetId );
```

What do you think?